lib/Formatting/Interfaces.ml

```diff
@@ -0,0 +1,8 @@
+module type CONVERTIBLE = sig
```

Comment:
Minor thing but I find `CONVERTIBLE` a bit too generic. It maybe fine in the context of this library and formatting, maybe it's just me.

But I would prefer `TOKENIZABLE` or even `TO_ELEMENT` or `ELEMENTABLE` but naming is generally hard.

Reply:
I was thinking of doing it the Rust `into` style but I figured that it would be too overkill
But yeah I think I'll use `TO_ELEMENT` ; even though the name doesn't matter that much (it is really rarely used by the users of Ansifmt) I am not satisfied with `CONVERTIBLE` either